_data/collection-index.yml

```
@@ -568,6 +568,11 @@
   contact: https://github.com/schlich/cst-devcontainer-feature/issues
   repository: https://github.com/schlich/cst-devcontainer-feature
   ociReference: ghcr.io/schlich/cst-devcontainer-feature
+- name: Rye
+  maintainer: Ty Schlichenmeyer
+  contact: https://github.com/schlich/devcontainer-features/issues
+  repository: https://github.com/schlich/devcontainer-features
+  ociReference: ghcr.io/schlich/devcontainer-features/rye
```

```suggestion
  ociReference: ghcr.io/schlich/devcontainer-features
```

Hi 👋 

The `ociReference` here refers to the collection metadata artifact instead of the individual Features. Hence, the author needs to only add a single entry irrespective of the number of Features that might exist in their repository.

Can you help update this? thanks!